include/caf/ble_smp.rst

```diff
@@ -0,0 +1,41 @@
+.. _caf_ble_smp:
+
+CAF: Simple Management Protocol module
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |smp| of the :ref:`lib_caf` (CAF) is responsible for performing a Device Firmware Upgrade (DFU) over Bluetooth LE.
+
+Configuration
+*************
+
+To use the module, you must enable the following Kconfig options:
```

Comment:
I would add information here that we actually need to enable and configure Bluetooth in CAF

Reply:
Without ble_state module enabled, the SMP would never initialize

Reply:
dependency in Kconfig would be sufficient?

Reply:
Will add this info together with dependency.

Reply:
In general we described the dependencies in documentation too. We could do it here aswell. @greg-fer, what's your opinion?